### PR TITLE
Make Logger stdout configurable using env variables

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | You may want to change the output path using the environment variable instead of updating the configuration files. Use `LOG_STDOUT` to define a new Logger output.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Setup the LOG_STDOUT environment variable and check if the logger output has been updated. IMHO, @Quetzacoalt91 is the best one to review this one.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10901)

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ parameters:
     translator.class: PrestaShopBundle\Translation\Translator
     translator.data_collector: PrestaShopBundle\Translation\DataCollectorTranslator
     admin_page: "%kernel.root_dir%/../src/PrestaShopBundle/Resources/views/Admin"
+    log_stdout: '%env(LOG_STDOUT)%'
+    env(LOG_STDOUT): "%kernel.logs_dir%/%kernel.environment%.log"
```
Comment (PierreRambaud) [inline app/config/config.yml]: `LOG_STDOUT` is too generic for the system. Should be `PS_LOG_STDOUT` instead

Comment (mickaelandrieu) [reply app/config/config.yml]: yes, but I need to keep `PS_*` available for ... configuration keys later. How about `_PS_LOG_STDOUT` ?

Comment (PierreRambaud) [reply app/config/config.yml]: And more, because it's not only an stdout log (stderr too), `PS_LOG_OUTPUT` can be used?

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
@Quetzacoalt91  you can try too =)

**Comment by Quetzacoalt91** [comment]
All good, I tried with this configuration.

```yml
version: '2'

volumes:
    db-data:

services:
    mysql:
        image: mysql:5.6
        ports:
            - "3306"
        volumes:
            - db-data:/var/lib/mysql
        environment:
            MYSQL_ROOT_PASSWORD: password
            MYSQL_DATABASE: prestashop
    apache:
        image: prestashop/prestashop-git:7.1
        environment:
            PS_INSTALL_AUTO: 1
            DB_PASSWD: password
            DB_NAME: prestashop
            DB_SERVER: mysql
            PS_DOMAIN: localhost:8001
            PS_FOLDER_INSTALL: install-dev
            PS_FOLDER_ADMIN: admin-dev
            _PS_LOG_OUTPUT_: "php://stdout"
        ports:
            - "8001:80"
```

The container log is now filled with additional details.

```
[2018-10-08 16:29:49] doctrine.DEBUG: SELECT t0.id_tab AS id_tab_1, t0.id_parent AS id_parent_2, t0.position AS position_3, t0.module AS module_4, t0.class_name AS class_name_5, t0.active AS active_6, t0.hide_host_mode AS hide_host_mode_7, t0.icon AS icon_8 FROM ps_tab t0 WHERE t0.class_name = ? LIMIT 1 ["AdminDashgoals"] []
[2018-10-08 16:29:56] doctrine.DEBUG: SELECT t0.id_tab AS id_tab_1, t0.id_parent AS id_parent_2, t0.position AS position_3, t0.module AS module_4, t0.class_name AS class_name_5, t0.active AS active_6, t0.hide_host_mode AS hide_host_mode_7, t0.icon AS icon_8 FROM ps_tab t0 WHERE t0.class_name = ? LIMIT 1 ["AdminLinkWidget"] []
[2018-10-08 16:30:03] doctrine.DEBUG: SELECT t0.id_tab AS id_tab_1, t0.id_parent AS id_parent_2, t0.position AS position_3, t0.module AS module_4, t0.class_name AS class_name_5, t0.active AS active_6, t0.hide_host_mode AS hide_host_mode_7, t0.icon AS icon_8 FROM ps_tab t0 WHERE t0.class_name = ? LIMIT 1 ["AdminWelcome"] []
```

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
As said to @mickaelandrieu, I'm not confortable with the use of `_PS_[...]` env var, as they differ from the existing format.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
No need QA here. Thanks @mickaelandrieu

**Pull request merged**